src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -100,14 +100,14 @@ std::string postEntity
     }
   }
 
-  // Default value for status code: SccCreated
+  // Default value for status code: SccNoContent
   if ((ciP->httpStatusCode == SccOk) || (ciP->httpStatusCode == SccNone) || (ciP->httpStatusCode == SccCreated))
```

Comment:
> that depends on whether any function sets ciP->httpStatusCode to SccNoContent before this moment in execution

Mmm... good point. I have assumed that "any function" = "the one in mongoBackend" but some other processing could be done in the middle. The whole logic of postEntities (and invoked functions) should be reviewed to check how ciP->hattpStatusCode is being manipulated.